certbot/tests/storage_test.py

```diff
@@ -19,6 +15,12 @@
 from certbot.compat import os
 import certbot.tests.util as test_util
 
+try:
+    import mock
+except ImportError: # pragma: no cover
+    from unittest import mock
```

Comment:
#8630 should already have dealt with this

Reply:
Read https://github.com/certbot/certbot/issues/7886#issue-596108651 if you're curious why things were done this way. Changing this to using `unittest.mock` breaks things when our public test utilities continue to use 3rd party `mock` if its installed.

Reply:
I believe the question is answered.